src/App.js

```diff
@@ -18,12 +18,6 @@ export default function App() {
       <form class="App-login">
         <FormGroup>
           <FormLabel>Email</FormLabel>
-          <FormControl 
```

Comment:
why did you removed it? we need it!

Reply:
just a text, don't piss off!

Reply:
please revert it back and send another pr.